src/cmd/ship.go

```diff
@@ -204,7 +212,11 @@ func getShipStepList(config shipConfig, repo *git.ProdRepo) (result steps.StepLi
 }
 
 func getCanShipWithDriver(branch, parentBranch string, driver drivers.CodeHostingDriver) (result drivers.PullRequestInfo, err error) {
-	if !git.HasRemote("origin") {
+	hasOrigin, err := prodRepo.Silent.HasRemote("origin")
```

Comment:
Thoughts on passing hasOrigin into this function since its already computed once before in the calling function?

Reply:
This function already has 3 input and 2 output arguments, and Git remotes are being cached so getting them is a cheap operation. Might not be worth it here.